CHANGELOG.md

```diff
@@ -29,6 +29,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   The `messaging.message_id` and `messaging.kafka.partition` attributes are now not set if a message was not processed. (#754) (#755) (#881)
 - Fix `otelsarama.WrapAsyncProducer` so that the messages from the `Errors` channel contain the original `Metadata`. (#754)
 
+### Added
+
+- Add `WithoutSubSpans`, `WithRedactedHeaders`, `WithoutHeaders`, and `WithInsecureHeaders` options for `otelhttptrace.NewClientTrace`. (#879)
+
+### Changed
+
+- `otelhttptrace.NewClientTrace` now redacts known sensitive headers by default. (#879)
+
```

Comment:
Please move this section up under `Unreleased` since this change was not included in `v0.22.0`.

Reply:
Updated, that was caused by the git automatic merge since I had not rebased since the latest release.